runtimes/sklearn/mlserver_sklearn/sklearn.py

```diff
@@ -51,14 +52,27 @@ async def predict(self, payload: InferenceRequest) -> InferenceResponse:
 
     def _check_request(self, payload: InferenceRequest) -> InferenceRequest:
         if not payload.outputs:
+            found_predict_fn = False
+            if self.settings.parameters:
+                print(self.settings)
+                if self.settings.parameters.extra:
+                    if PREDICT_FN_KEY in self.settings.parameters.extra:
+                        payload.outputs = [
+                            RequestOutput(
+                                name=self.settings.parameters.extra[PREDICT_FN_KEY]
+                            )
+                        ]
+                        found_predict_fn = True
             # By default, only return the result of `predict()`
-            payload.outputs = [RequestOutput(name=PREDICT_OUTPUT)]
+            if not found_predict_fn:
+                payload.outputs = [RequestOutput(name=PREDICT_OUTPUT)]
         else:
-            for request_output in payload.outputs:
+            for request_output in payload.outputs:  # type: ignore
```

Comment:
Do you know why does the linter complains about this line?

Reply:
removed and seems ok now. Not sure why was failing before